File: pre-contest/a_42/sol.cs
using System;

class a_42 {
    public static void Main() 
    {
        string Nstr = Console.ReadLine();
        int N = int.Parse(Nstr);
        
        if (N < 42)
            Console.Write("More");
        else if (N > 42)
            Console.Write("Less");
        else if (N == 42)
            Console.Write("Bingo!");
    }
}

File: pre-contest/a_42/sol/sol.rs
use std::io;
use std::cmp::Ordering;

fn main() {
    // I don't check errors since we are in a contest.
    // Moreover, using the crate text_io can reduce
    // all this code to a simple read!(), but it's not
    // available during contests.
    let mut line = String::new();
    io::stdin().read_line(&mut line).unwrap();
    let n = line.trim().parse::<i32>().unwrap();

    // I could have used a standard if construction, but
    // where is the fun in that? ;)
    println!("{}", match 42.cmp(&n) {
        Ordering::Greater => "More",
        Ordering::Less => "Less",
        Ordering::Equal => "Bingo!",
    });
}

